spec/support/shared_examples/configurable.rb

```diff
@@ -224,20 +224,20 @@
           end
         end
 
-        context 'when the inherited settings are modified' do
+        context 'when the inherited settings are modified ' do
           before do
             klass.setting :dsn
             klass.configure do |config|
               config.dsn = 'jdbc:sqlite:memory'
             end
-
-            subclass.setting :db
```

Comment:
Does this test not pass is this is executed before `klass.configure` is called above?